app/src/main/java/com/babylon/wallet/android/presentation/accessfactorsources/deriveaccounts/DeriveAccountsViewModel.kt

```diff
@@ -206,15 +212,15 @@ class DeriveAccountsViewModel @Inject constructor(
                 accessFactorSourcesIOHandler.setOutput(
                     output = AccessFactorSourcesOutput.DerivedAccountsWithNextDerivationPath(
                         derivedAccounts = derivedAccounts,
-                        nextDerivationPathOffset = indicesToScan.last() + 1u
+                        nextDerivationPathIndex = HdPathComponent.init(globalKeySpace = indicesToScan.last().indexInGlobalKeySpace + 1u)
```

Comment:
... also here `last()` was used. That is why I think `LinkedHashSet` should be used.